src/Configuration/Configuration.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+declare(strict_types=1);
+
+namespace MyParcelCom\Integration\Configuration;
+
+use Illuminate\Support\Arr;
+use MyParcelCom\Integration\Configuration\Properties\PropertyCollection;
+use MyParcelCom\Integration\ProvidesJsonAPI;
+
+class Configuration implements ProvidesJsonAPI
```

Comment:
Since the object does not really render as json API we could instead just implement the Laravel Responsible interface and construct a JsonResponse object with the payload.